Moving more extensions packages into aspnetcore (#27919)
dotnet/aspnetcore: src/Caching/README.md

# Caching

> This area contains distributed cache implementations for Microsoft SQL Server, and Redis. See documentation at https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests#use-polly-based-handlers.

Reviewer: nit
```suggestion
This area contains distributed cache implementations for Microsoft SQL Server and Redis. See documentation at <https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-requests#use-polly-based-handlers>.
```

## Description

> These packages are not part of the ASP.NET Core shared framework but are shipped as an individual package. This packages has a third party dependencies on Polly and Polly.Extensions.Http.

Reviewer: ```suggestion
These packages are not part of the ASP.NET Core shared framework but are shipped as an individual package. These packages have a third party dependencies on Polly and Polly.Extensions.Http.
```

> Caching

Reviewer: Suggest following up to use the template for area-level README files from #27561
